src/styles/tailwind.css

```
@@ -1,3 +1,34 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@layer base, starlight, theme, components, utilities;
+
+@import '@astrojs/starlight-tailwind';
+@plugin '@tailwindcss/typography';
+@config '../../tailwind.config.ts';
+@import 'tailwindcss/theme.css' layer(theme);
```

Stylelint (17.12.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)